Rural Art Template/index.css

```diff
@@ -6,6 +6,7 @@
 }
 
 .nav {
+    width: 100%;
   background-color: rgb(182, 88, 85);
 }
 
```

Comment:
@Anirban-1490 update ur branch 
these files should not be there

Reply:
yes, sorry i accidently pushed it , even though i was using a diff branch when doing the commit for the rural art issue. now i can't figure out what to do . any help please